doc/scenario.md

```diff
@@ -2,15 +2,17 @@
 
 All files related to scenarios should be placed in the `scenarios` directory.
 
-A scenario specification is an [Erlang](http://www.erlang.org/) module that
-exposes two callback functions:
-- ``init/0`` - called only once per test run, at the very beginning.
-It can be used for setting up initial (global) state: metrics, database
-connections, etc.
-- ``start/1`` - describes the actual scenario and is executed for
+A scenario specification is an [Erlang](https://www.erlang.org/) module that implements
+the `amoc_scenario` behaviour.
+It has to export two callback functions:
+- `init/0` - called only once per test run on every node, at the very beginning.
+It can be used for setting up initial (global) state: metrics, database connections, etc.
+It can return an `ok` or a tuple `{ok, State}` from which the `State` may be passed to every user.
+- `start/1` or `start/2` - describes the actual scenario and is executed for
 each user, in the context of that user's process.
-It takes one argument, which is the given user's unique integer id.
-When the function returns, it is executed again after some delay (60 seconds by default).
+The first argument is the given user's unique integer id.
+The second, which is optional, is the state, as returned from `init/0` function.
+When `start` function returns, it is executed again after some delay (60 seconds by default).
```

Comment:
> When the `start` function returns, [...]

or

> When `start` returns

(The first option is with  a determinate article used to specify which "function", the start function; the second option needs no article because in that one we use the name "start" as a proper name, which doesn't need an article).